packages/react/src/utils/felaExpandCssShorthandsPlugin.ts

```diff
@@ -1,52 +1,56 @@
-import * as _ from 'lodash'
-import expand from './cssExpandShorthand'
-import * as _memoize from 'fast-memoize'
-
-// `fast-memoize` is a CJS library, there are known issues with them:
-// https://github.com/rollup/rollup/issues/1267#issuecomment-446681320
-const memoize = (_memoize as any).default || _memoize
-
-// _.camelCase is quite fast, but we are running it for the same values many times
-const camelCase = memoize(_.camelCase)
-
-const handledCssPropsMap = {
-  font: 'font',
-  padding: 'padding',
-  margin: 'margin',
-  border: 'border',
-  borderWidth: 'border-width',
-  borderStyle: 'border-style',
-  borderColor: 'border-color',
-  borderTop: 'border-top',
-  borderRight: 'border-right',
-  borderBottom: 'border-bottom',
-  borderLeft: 'border-left',
-  borderRadius: 'border-radius',
-  background: 'background',
-  outline: 'outline',
+import { ICSSInJSStyle } from '@fluentui/styles'
+import * as CSS from 'csstype'
+import { expandProperty } from 'inline-style-expand-shorthand'
+
+// https://jsperf.com/array-indexof-vs-object-key-lookup2/12
+const handledCssProps: Partial<Record<keyof CSS.Properties, true>> = {
+  // 'font', Oops, is not supported by inline-style-expand-shorthand
```

Comment:
Please add **BREAKING CHANGE** changelog entry

Reply:
Added and updated PR's description